src/common/snippets/include/snippets/op/fma.hpp

```diff
@@ -0,0 +1,38 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include "ngraph/op/op.hpp"
+
+namespace ngraph {
+namespace snippets {
+namespace op {
+
+/**
+ * @interface FMA
+ * @brief Fused Multiply Add
+ * @ingroup snippets
+ */
+class FMA : public ngraph::op::Op {
```

Comment:
It's probably better to spell it explicitly FusedMultiplyAdd (or FusedMulAdd) to align with other ops like blockedparameter or broadcastmove, moreover it would be clearer for other plugins who might not have explicit fma instruction in their hardware opset. The same applies to filename.

Reply:
All related classes and files were renamed, thanks for the suggestion.